addons/web/static/src/core/file_upload/file_upload_progress_record.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="web.FileUploadProgressKanbanRecord" owl="1">
```

Comment:
not sure why do need to reference kanban view here. it would be better if this component was completely independant

Reply:
Keep in mind that this change is meant to replace what was previously available, maybe the file should be moved closer to `kanban` instead?